code/mission/missionparse.h

```diff
@@ -149,6 +150,7 @@ typedef struct mission {
 	int	command_persona;
 	char command_sender[NAME_LENGTH];
 	int debriefing_persona;
+	traitor_override *trtr_override;
```

Comment:
I'd suggest using `traitor_override_t` for the type and `traitor_override` for the field.  This is how it's done in several other places, e.g. `ai_profile_t`, `asteroid_t`, `opengl_shader_t`.  This would avoid the need to abbreviate the field.

Reply:
The `_t` token should be the type name, as the t stands for type.